process/PC_PL_Template_Markdown.md

```diff
@@ -1,23 +1,25 @@
 # OpenHW Project Concept And Project Launch Markdown Template: Instructions
-*This template is divided into two parts:*
-- *The **Project Concept (PC)** required fields are shown in the first part.*
-- *The additional **Project Launch (PL)** required and optional fields are shown in the second part.*
+<!--- This template is divided into two parts:
```

Comment:
The word "Instructions" in the header looks like an orphan.  Maybe something like this:
```
# OpenHW Project Concept And Project Launch Markdown Template
<!---
Instructions for use: this template is divided into two parts:
....
```